x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/GetFlamegraphResponse.java

```diff
@@ -23,8 +23,8 @@
 public class GetFlamegraphResponse extends ActionResponse implements ChunkedToXContentObject {
     private final int size;
     private final double samplingRate;
-    private final int selfCPU;
-    private final int totalCPU;
+    private final long selfCPU;
```

Comment:
The type of this field (and others) change because due to the different aggregation we use now `Bucket#getDocCount()` to retrieve document counts and that method returns a `long`.